File: Minimum Element After Replacement With Digit Sum.cpp
class Solution {
public:
    int minElement(vector<int>& nums) {
        int min=nums[0];
        for(int i=0;i<nums.size();i++){
            int sum=0;
            int temp=nums[i];
            while(temp>0){
                int rem= temp%10;
                sum+=rem;
                temp=temp/10;
            }
            nums[i]=sum;
            if(nums[i]<min){
                min=nums[i];
            }
        }
        return min;
    }
};

File: Leetcode-Problems/Minimum Element After Replacement With Digit Sum.java
class Solution {
    public int minElement(int[] nums) {
        int ans = Integer.MAX_VALUE;
        for (int i : nums) {
            int sum = 0;
            while (i > 0) {
                sum += i % 10;
                i /= 10;
            }
            ans = Math.min(ans, sum);
        }
        return ans;
    }
}